modin/backends/pandas/parsers.py

```diff
@@ -314,6 +314,8 @@ def parse(fname, **kwargs):
             to_read = header + bio.read(end - start)
             bio.close()
             pandas_df = pandas.read_fwf(BytesIO(to_read), **kwargs)
+            if pandas_df.index.name == "Unnamed: 0":
```

Comment:
what if the index name is `Unnamed: 15` or something similar?..

also why're you changing it in `read_fwf` instead of `read_csv` (and not in generic `_read` or smth - I doubt it's specific to only one text format)